### PR TITLE
fix: Ga4gh tes bugfixes

### PR DESCRIPTION
**Pull request by svedziok**
### Description

This PR contains two bugfixes
- Adds missing py-tes package, which is required for GA4GH job submission
- Resolve bug with remote log or benchmarkfiles

### QC

* [ x] The PR contains a test case for the changes or the changes are already covered by an existing test case.
* [ x] The documentation (`docs/`) is updated to reflect the changes or this is not necessary (e.g. if the change does neither modify the language nor the behavior or functionalities of Snakemake).

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -76,6 +76,7 @@
         "filelock",
         "stopit",
         "tabulate",
+        "py-tes",
```
Comment (johanneskoester) [inline setup.py:79]: I don't think this needs to be a required dependency. But the docs should probably say that py-tes needs to be installed.

Comment (johanneskoester) [reply setup.py:79]: And the import statement [here](https://github.com/snakemake/snakemake/pull/1127/files#diff-a1865e2681ce5d0b775d62c336fe1ac260c6855d91715e5bb033a2babaae830dR40) should be wrapped in a try-except with a `WorkflowError("Unable to import Python package tes. TES backend requires py-tes to be installed. Please install py-tes, e.g. via Conda or Pip.")`

**Review by johanneskoester — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (svedziok) [reply setup.py:79]: Ok, I added a try-except statement in the __init__ function. I will also a note at the documentation

**Review by johanneskoester — APPROVED** [review]

**Comment by johanneskoester** [comment]
Thank you!

**Pull request merged**